AssetManagementTestProject/TestCases/US302_LoginTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.Services;
+using AssetManagementTestProject.TestData;
+using AssetManagementTestProject.TestSetup;
+using NUnit.Framework;
+
+namespace AssetManagementTestProject;
+
+[TestFixture]
+public class US302_LoginTest : NUnitWebTestSetup
+{    
+    protected ChangePassword1stTimePage? ChangePw1stTime;
+
+
+    [TestCase(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD_HN)]
+    [TestCase(Constant.STAFF_USERNAME, Constant.STAFF_PASSWORD)]
+    public void TC01_UserLoginSuccess(string username, string password)
+    {
+        LoginPage?.Login(username, password);
+        DriverBaseAction?.WaitToBeVisible(HomePage.HeaderHomePage);
+        Asserter?.AssertElementIsDisplayed(HomePage.HeaderHomePage);
+    }
+    [Test]
+    public void TC02_UserAskedChangePasswordFirstTime()
+    {
+        AuthorizationService = new AssetManagementAPIServices();
+        Token = AuthorizationService.ReturnLoginToken(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD_HN);
+        APIService = new AssetManagementAPIServices();
```

Comment:
Move this param to the parent as a protected property

Reply:
I was thinking about the putting this in the WebTestSetup a few days ago, but if I do so then every test case will create a new user with API? If so, that will be unnecessary for test cases such as TC01 (Login with ADMIN password)